src/components/Footer.js

```diff
@@ -5,16 +5,16 @@ const Footer = (props) => (
     <footer id="footer">
         <section>
             <h2>We Are Open Source</h2>
-            <p>Gastown Gang operates transparently. We welcome all contributions!</p>
+            <p>Cambie Collective operates transparently. We welcome all contributions!</p>
             <ul className="actions">
-                <li><a href="https://github.com/gastowngang/gastowngang.github.io" className="button">Join us!</a></li>
+                <li><a href="https://github.com/cambiecollective/cambiecollective.github.io" className="button">Join us!</a></li>
             </ul>
         </section>
         <section>
             <h2>Find us on the internet</h2>
             <ul className="icons">
-                <li><a href="https://twitter.com/gastowngang" className="icon fa-twitter alt"><span className="label">Twitter</span></a></li>
-                <li><a href="https://github.com/gastowngang/gastowngang.github.io" className="icon fa-github alt"><span className="label">GitHub</span></a></li>
+                <li><a href="https://twitter.com/cambiecollective" className="icon fa-twitter alt"><span className="label">Twitter</span></a></li>
```

Comment:
As the todo in the PR body mentioned, the twitter did need updating :)

For now it's `cambiecollectiv` without the "e". Thanks to Twitter's handle length limits.

Reply:
TIL: My full name on Twitter is _exactly_ the max length. 😅